src/components/mgt-agenda/mgt-agenda.ts

```diff
@@ -204,10 +230,25 @@ export class MgtAgenda extends MgtTemplatedComponent {
     return html`
       <div class="agenda${this._isNarrow ? ' narrow' : ''}${this.groupByDay ? ' grouped' : ''}">
         ${this.groupByDay ? this.renderGroups(events) : this.renderEvents(events)}
+        ${this._isLoadingEvents ? this.renderLoading() : html``}
```

Comment:
I don't fully understand what `_isLoadingEvents` is doing. Why use it on top of what `isLoadingState` does?

Reply:
I knew I forgot to comment on something :)

`isLoadingState` is true while the loadState method promise completes. The issue with that is that the UI will not update until all events load - that could take few seconds depending on the number of events. I want to show the events as they they are fetched from the graph. I added `_isLoadingEvents` to signal that I'm loading events and to show any events that might have loaded already but also show the loading view at the end of the list to indicate that events are still loading

Reply:
Gotcha! Looks good to me. Just wanted to clarify.